### PR TITLE
Reenable some catalogue solution tests

### PR DESCRIPTION
**Pull request by PhysicsJon**

### CODE REVIEW

**Review by gavinwakeley1 — COMMENTED** [review]
Azure bit looks fine - will leave to someone else to approve the code changes

File: src/OrderFormAcceptanceTests.TestData/Order.cs [hunk 1]
```diff
@@ -64,6 +65,12 @@ public Order Retrieve(string connectionString)
             return SqlExecutor.Execute<Order>(connectionString, query, this).Single();
         }
 
+        public IEnumerable<int> GetContactIdsForOrder(string connectionString)
+        {
+            var query = "(SELECT OrganisationContactId from [dbo].[Order] WHERE OrderId=@orderId) UNION (SELECT SupplierContactId from [dbo].[Order] WHERE OrderId=@orderId)";
```
Comment (bjss-chris) [inline src/OrderFormAcceptanceTests.TestData/Order.cs:70]: Just a few minor formatting suggestions here:

1. Capitalize `FROM` to be consistent with other SQL statements?
2. The square brackets aren't required around `dbo`.
3. Spaces around `=` operator?
4. Add a terminating semicolon.

File: src/OrderFormAcceptanceTests.TestData/OrganisationInfo.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+namespace OrderFormAcceptanceTests.TestData
+{
+    public static class OrganisationInfo
+    {
+        public static void DeleteContactsForOrdersNoLongerInDb(string connectionString)
+        {
+            var query = "DELETE FROM dbo.Contact " +
+                "WHERE ContactId NOT IN (SELECT OrganisationContactId FROM dbo.[Order])" +
+                "OR ContactId NOT IN (SELECT SupplierContactId FROM dbo.[Order]";
```
Comment (bjss-chris) [inline src/OrderFormAcceptanceTests.TestData/OrganisationInfo.cs:11]: Would just recommend adding a terminating semicolon to the SQL statement.

**Review by bjss-chris — COMMENTED** [review]
Just left a couple of comments with the usual SQL formatting suggestions, but otherwise looks fine.

**Review by bjss-chris — APPROVED** [review]
👍

**Pull request merged**